### ISSUE TITLE
"Cannot read property 'sessionId' of undefined" after 

### ISSUE THREAD

**Issue by donaldpipowitch** [webdriverio/webdriverio#775]
Hi,
I called `wdio` and created a nice config. If I run `wdio` now I get this error:

````
/webdriverio/lib/utils/RequestHandler.js:134
            this.sessionID = body.sessionId;
                                 ^

TypeError: Cannot read property 'sessionId' of undefined
````

Selenium grid is running. I tried it with Protractor and Nightwatch, too.

**Comment by stoffeastrom** [comment]
@donaldpipowitch are you running selenium on another port than 4444? Had a similiar issue which turned out being that the `wdio.conf.js` port config was overwritten with the default port 4444

**Comment by donaldpipowitch** [comment]
We use `port: 1337,` and it is set in my `wdio.conf.js`.

**Comment by stoffeastrom** [comment]
And I guess you are running `3.2.2`. If that's the case try specifically install `3.2.1`. I think there was a regression introduced in https://github.com/webdriverio/webdriverio/commit/9fe0b5f4046c1117df02c11759b4b69dff72852a

**Comment by donaldpipowitch** [comment]
Yea, I run `3.2.2`. I'll try `3.2.1`. Thanks a lot!

**Comment by christian-bromann** [comment]
Sorry about that, will look into it asap

**Comment by christian-bromann** [comment]
Fixed and publish with `v3.2.3`
